library.js

```diff
@@ -1,44 +1,50 @@
 const inputUser = document.querySelector('.input-user');
 const createBookBtn = document.querySelector('.crt_book');
 const readBtn = document.querySelector('.btn-read');
+const form = document.querySelector('#form');
+
```

Comment:
most of the changes are because of my prettier configs sorry there are only 2 main changes, here